### PR TITLE
update digga

### PR DESCRIPTION
**Pull request by blaggacao**
- rm: use of patched nix for nixos 21.11 enabled digga
- fix: rm hm enableFlakes
- imp: move follows responsibility downstream

### CODE REVIEW

**Comment by blaggacao** [comment]
This is the necessary companion of https://github.com/divnix/digga/pull/115

File: flake.lock [hunk 1]
```diff
@@ -147,34 +144,50 @@
...
+        "treefmt": "treefmt"
       },
       "locked": {
-        "lastModified": 1634161492,
-        "narHash": "sha256-7OZz7DqAhO+3axY68xsMXAyD+b5gJgVXuKjslTSCM3Y=",
-        "owner": "divnix",
-        "repo": "digga",
-        "rev": "a55450a16d362b6e1c50bb4025aaa604b385d3ba",
+        "narHash": "sha256-rjVhPGMc58j8AE/FSmIrSyNSDN51o0Ooob0oRh/kq/U=",
+        "path": "/home/blaggacao/src/github.com/divnix/digga",
```
Comment (teutat3s) [inline flake.lock:167]: Intentionally left path here?

File: flake.lock [hunk 2]
```diff
@@ -147,34 +144,50 @@
...
+        "type": "path"
+      },
+      "original": {
+        "path": "/home/blaggacao/src/github.com/divnix/digga",
```
Comment (teutat3s) [inline flake.lock:171]: Ditto?

**Review by teutat3s — COMMENTED** [review]
LGTM, nice cleanup, only two small questions.

**Comment by benneti** [comment]
I think during this update https://github.com/divnix/devos/blob/main/overlays/various-flake-fixes.nix should be deleted as this attr is depreciated and not necessary anymore

File: flake.lock (hunk 1 above)
Comment (blaggacao) [reply flake.lock:167]: Oops ;-)

**Comment by Pacman99** [comment]
Is this ready to be merged? I think this should fix digga CI.

**Comment by Pacman99** [comment]
bors try

**Comment by Pacman99** [comment]
bors try

**Comment by Pacman99** [comment]
bors try

**Comment by Pacman99** [comment]
bors try

**Comment by Pacman99** [comment]
bors try

**Comment by Pacman99** [comment]
bors try

**Comment by Pacman99** [comment]
bors try

**Comment by Pacman99** [comment]
bors try

**Comment by Pacman99** [comment]
bors try

**Comment by Pacman99** [comment]
bors try

**Comment by Pacman99** [comment]
All the checks work locally on my system, I think this is an issue with the cache, so I'm going to go ahead and merge.

**Pull request merged**